tests/Amazon.CloudWatch.EMF.Tests/Logger/MetricsLoggerTests.cs

```diff
@@ -333,6 +334,23 @@ public void TestPutMetricWithUnit()
 
             Assert.Equal(expectedValue, metricDefinition.Values[0]);
             Assert.Equal(Unit.MILLISECONDS, metricDefinition.Unit);
+            Assert.Equal(StorageResolution.STANDARD, metricDefinition.StorageResolution);
+        }
+
+        [Fact]
+        public void TestPutMetricWithHighResolutionCheck()
```

Comment:
Can we also have a test case where user explicitly specifies storageResolution as 'STANDARD'. 
PutMetric(key, value, Unit.MILLISECONDS,StorageResolution.STANDARD)